test_suite/tests/test_annotations.rs

```diff
@@ -780,7 +780,7 @@ fn test_rename_enum() {
             Token::StructVariant {
                 name: "AliasEnum",
                 variant: "sailor_moon",
-                len: 3,
```

Comment:
This seems wrong. The struct still only has a length of 3,since three fields are emitted. Not 5 just because there are five field names and aliases.

Reply:
In the serde data model, the struct definitely has 5 fields (some of which are optional, some subsets of which are mutually exclusive).

The exact number of Rust fields that the deserialized Rust representation ends up storing doesn't seem relevant. If we wrote a Deserialize impl that stored none of them, it doesn't make a difference to the Deserializer. Or compare with how `#[serde(transparent)] struct Struct(i32);` deserializes using deserialize_i32, not deserialize_newtype_struct, even with the deserialized Rust representation `Struct` being a struct not an int -- that isn't relevant.